vm/evm.go

```diff
@@ -173,6 +173,8 @@ type EVM struct {
 	callGasTemp uint64
 	// state change & call stack tracer
 	tracer *Tracer
+
+	IsExecuteJP bool
```

Comment:
Is this requested to be exported? Are there any outside packages using this property?